src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -165,6 +166,7 @@
           maccatalyst-x64;
           maccatalyst-arm64;
           linux-s390x;
+          linux-ppc64le;
```

Comment:
These should be added to `Net70AppHostRids`/`Net70RuntimePackRids`, right? Do we really want to modify the set of architectures in .NET 6?

Reply:
I agree. This should be added to Net70AppHostRids/Net70RuntimePackRids